src/conditional-flow/boolean-conditions.js

```diff
@@ -3,7 +3,13 @@
 // if the value is false.
 function getResult (didPass) {
 
-  // TODO: write code in this function body to pass the tests
+  if (didPass==true) {
+    return "Well done, you passed!";
+  }
+
+  else {
```

Comment:
This is a rather pedantic comment but you don't need this `else` here. In your first if statement, you're using return. This will stop the execution of any code underneath it. As your first if statement is checking a boolean value, you know the only way that the first if statement isn't executed is if the variable if false, so you could just have the return statement.

Reply:
Ah ok! I did not know that! So I should have just written return "You did not pass" (or whatever the comment was?). No else needed?

Reply:
Yep! No else is required, because you terminate the function if your if statement. This means that the only way for the computer to execute code after that if statement is if the `didPass` variable is false, which is essentially what the else statement would have done.